packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/__tests__/LivenessCameraModule.test.tsx

```diff
@@ -507,6 +507,31 @@ describe('LivenessCameraModule', () => {
     ).toBeInTheDocument();
   });
 
+  it('should render hair check screen when isStart = true, should not render camera selector if only one camera', () => {
```

Comment:
Do we need to check if the hair check screen is rendered when isStart = true here when that's the purpose of the preceding test?